src/pages/NotFound.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+    <GqContainer>
+        <div>
+            <h1 class="mt-5">Sorry, this page does not exist</h1>
+            <router-link :to="{name: 'sign-in'}"
```

Comment:
or router-link with class btn or button with @click
now you get "button inside a"